rest_framework/serializers.py

```diff
@@ -346,6 +346,26 @@ class Serializer(BaseSerializer, metaclass=SerializerMetaclass):
         'invalid': _('Invalid data. Expected a dictionary, but got {datatype}.')
     }
 
+    def set_value(self, dictionary, keys, value):
```

Comment:
do we need any documentation update // adjustments for the new changes?

Reply:
I wouldn't think so, as the change is really just internal and does not affect how `set_value()` works. Would you want me to document the changes somewhere?